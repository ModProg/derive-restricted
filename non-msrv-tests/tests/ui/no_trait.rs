#[derive_where::derive_where("Clone")]
struct Test<T>(T);

fn main() {}
