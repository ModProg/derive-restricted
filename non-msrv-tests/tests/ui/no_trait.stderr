error: expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd, Zeroize
 --> tests/ui/no_trait.rs:1:30
  |
1 | #[derive_where::derive_where("Clone")]
  |                              ^^^^^^^
